src/Engine/Component/AnimationController.cpp

```diff
@@ -88,8 +88,8 @@ void AnimationController::Animate(float deltaTime, Animation::AnimationControlle
     Animation::AnimationClip::Animation* anim = action->animationClip->animation;
     std::size_t size = skeleton->skeletonBones.size() > anim->numBones ? anim->numBones : skeleton->skeletonBones.size();
 
-    anim->currentFrame += deltaTime * 1.0f;
-    if (anim->currentFrame > 15.0f) {
+    anim->currentFrame += deltaTime * 30.0f;
+    if (anim->currentFrame > 119.0f) {
```

Comment:
Add comment to explain the 119.0f?

Reply:
This is hardcoded, I'm changing this to a non hardcoded value. And feel this code lacks comments overall so will be adding some more comments.